hub/integrations/tests/test_pytorch.py

```diff
@@ -82,6 +131,22 @@ def test_pytorch_small(ds):
             batch["image2"].numpy(), (i) * np.ones((1, 12, 12))
         )
 
+    sub_dls3 = sub_ds3.pytorch(num_workers=2, batch_size=1, shuffle=True)
+    all_values = []
+    for i, batch in enumerate(sub_dls3):
+        value = batch["image"].numpy()[0][0][0]
+        value2 = batch["image2"].numpy()[0][0][0]
+        assert value == value2
+        all_values.append(value)
+        np.testing.assert_array_equal(
+            batch["image"].numpy(), value * np.ones((1, 10, 10))
+        )
+        np.testing.assert_array_equal(
+            batch["image2"].numpy(), value2 * np.ones((1, 12, 12))
+        )
+
+    assert set(all_values) == set(range(5))
```

Comment:
yea, these test methods are getting extremely long. i recommend creating helper functions to generate these datasets and then have separate functions for the shuffling tests that call those dataset population helpers